README.md

```diff
@@ -85,8 +85,8 @@ Only tested code will be accepted. Please follow fix the style guide.
 
```

Comment:
I did actually! (Well, `bro-code`, there was no `broc-code` - that's why I changed it in the `README.md`.) It didn't seem to have changed anything.